src/modules/fancyzones/lib/trace.cpp

```diff
@@ -77,7 +77,13 @@ void Trace::SettingsChanged(const Settings& settings) noexcept
         TraceLoggingBoolean(settings.zoneSetChange_flashZones, "FlashZonesOnZoneSetChange"),
         TraceLoggingBoolean(settings.zoneSetChange_moveWindows, "MoveWindowsOnZoneSetChange"),
         TraceLoggingBoolean(settings.overrideSnapHotkeys, "OverrideSnapHotKeys"),
-        TraceLoggingWideString(settings.zoneHightlightColor.c_str(), "ZoneHighlightColor"));
+        TraceLoggingBoolean(settings.appLastZone_moveWindows, "MoveWindowsToLastZoneOnAppOpening"),
+        TraceLoggingBoolean(settings.use_cursorpos_editor_startupscreen, "UseCursorPosOnEditorStartup"),
+        TraceLoggingWideString(settings.zoneHightlightColor.c_str(), "ZoneHighlightColor"),
+        TraceLoggingInt32(settings.zoneHighlightOpacity, "ZoneHighlightOpacity"),
+        TraceLoggingWideString(settings.editorHotkey.get_json().Stringify().c_str(), "HotkeyObject"),
+        TraceLoggingWideString(settings.excludedApps.c_str(), "ExcludedApps")
```

Comment:
We may not be allowed to trace the excludedApps app list for privacy reasons. So for now, instead of logging the list, let's just log how many apps are listed.